### DISCUSSION TITLE
Overruling Device Types Questions

### DISCUSSION THREAD

**Discussion by Hedrakk** [milo526/homebridge-tuya-web#217] [Q&A]
Hi,

I don't know much about Homebridge/Hoobs as I'm getting started with this.

First of all, thanks for this nice pluggin. It saved me a lot of money by using Homekit certified Smart Devices. Most of my Smart devices are supported by Tuya.

So here's my question: I'm trying to use "Overruling Device Types" to correct the type of some of my devices. The issue I have is that I can seem to make it work for more than one device... If I try have put more than one, when I save the changes on HOOBS, if I check back my Tuya Web settings, only one is there or none at all (I guess this happens when I try a bad format of "Overruling Device Types" definition).

Can you show me an example of the "Overruling Device Types" settings that would set more than one device?

Thanks.

**Comment by milo526** [comment]
Multiple device types can be configured as follows:

```json
{
  "platform": "TuyaWebPlatform",
  "name": "TuyaWebPlatform",
  "options": {
    ...some options
  },
  "defaults": [
    {
      "id": "<device 1 name or id>",
      "device_type": "<desired device 1 type>"
    },
   {
      "id": "<device 2 name or id>",
      "device_type": "<desired device 2 type>"
    }
  ]
}
```
